connector-sdk/jackson-datatype-feel/src/test/java/io/camunda/connector/feel/jackson/FeelFunctionDeserializerTest.java

```diff
@@ -201,7 +201,7 @@ void feelFunctionDeserialization_contextAware_mergedWithInput() throws IOExcepti
   }
 
   @Test
-  void feelFunctionDeserlization_contextAware_knowsJava8Time() throws IOException {
+  void feelFunctionDeserialization_contextAware_knowsJava8Time() throws IOException {
```

Comment:
Do these changes come from the autoformatting or are they intentional?

Reply:
I just fixed the typo here